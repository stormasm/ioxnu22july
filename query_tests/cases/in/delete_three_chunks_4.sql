-- Demonstrate soft deleted rows will not be return to queries
-- IOX_SETUP: ThreeDeleteThreeChunks

----------
SELECT * from cpu where bar >= 1.0 order by bar, foo, time;

SELECT foo from cpu where bar >= 1.0 order by foo;

SELECT time, bar from cpu where bar >= 1.0 order by bar, time;

SELECT * from cpu where foo = 'you' order by bar, foo, time;

SELECT min(bar) as mi, max(time) as ma from cpu where foo = 'you' order by mi, ma;
