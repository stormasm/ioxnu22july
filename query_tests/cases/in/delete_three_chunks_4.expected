-- Test Setup: ThreeDeleteThreeChunks
-- SQL: SELECT * from cpu where bar >= 1.0 order by bar, foo, time;
+-----+-----+--------------------------------+
| bar | foo | time                           |
+-----+-----+--------------------------------+
| 1   | me  | 1970-01-01T00:00:00.000000040Z |
| 1   | me  | 1970-01-01T00:00:00.000000042Z |
| 1   | me  | 1970-01-01T00:00:00.000000062Z |
| 3   | you | 1970-01-01T00:00:00.000000070Z |
| 4   | me  | 1970-01-01T00:00:00.000000050Z |
| 5   | me  | 1970-01-01T00:00:00.000000060Z |
| 7   | me  | 1970-01-01T00:00:00.000000080Z |
+-----+-----+--------------------------------+
-- SQL: SELECT foo from cpu where bar >= 1.0 order by foo;
+-----+
| foo |
+-----+
| me  |
| me  |
| me  |
| me  |
| me  |
| me  |
| you |
+-----+
-- SQL: SELECT time, bar from cpu where bar >= 1.0 order by bar, time;
+--------------------------------+-----+
| time                           | bar |
+--------------------------------+-----+
| 1970-01-01T00:00:00.000000040Z | 1   |
| 1970-01-01T00:00:00.000000042Z | 1   |
| 1970-01-01T00:00:00.000000062Z | 1   |
| 1970-01-01T00:00:00.000000070Z | 3   |
| 1970-01-01T00:00:00.000000050Z | 4   |
| 1970-01-01T00:00:00.000000060Z | 5   |
| 1970-01-01T00:00:00.000000080Z | 7   |
+--------------------------------+-----+
-- SQL: SELECT * from cpu where foo = 'you' order by bar, foo, time;
+-----+-----+--------------------------------+
| bar | foo | time                           |
+-----+-----+--------------------------------+
| 3   | you | 1970-01-01T00:00:00.000000070Z |
+-----+-----+--------------------------------+
-- SQL: SELECT min(bar) as mi, max(time) as ma from cpu where foo = 'you' order by mi, ma;
+----+--------------------------------+
| mi | ma                             |
+----+--------------------------------+
| 3  | 1970-01-01T00:00:00.000000070Z |
+----+--------------------------------+
